arcade/experimental/video_cv2.py

```diff
@@ -47,8 +51,10 @@ def __init__(self):
         # Configure videoFrame sampler to read from texture channel 0
         self.program["videoFrame"] = 0
 
-        # Open the video (can also read from webcam)
-        self.video = cv2.VideoCapture("C:/Users/efors/Desktop/BigBuckBunny.mp4")
+        # Used this because it will throw SIGSEGV when passed a Path like object, which is not very descriptive.
+        if not issubclass(type(path), str):
+            raise TypeError(f"The path is required to be a str object and not a {type(path)} object")
```

Comment:
Just resolve the resource usng arcade.resources.resolve_resource_path and allow str and Path.